cmd/scsd/scsd.go

```diff
@@ -66,10 +66,10 @@ const (
 	LOGLVL_PANIC = "PANIC"
 )
 
-var appParams = opParams{LogLevel: LOGLVL_ERROR,
+var appParams = opParams{LogLevel: LOGLVL_TRACE,
```

Comment:
Why did you set this to trace?

Reply:
Somehow you did not see the last commits - that was changed back to ERROR